site/_posts/2020-09-10-building-profile-portal-with-flask-oauth-apis.md

```diff
@@ -597,6 +626,8 @@ def homepage():
     client_response = fusionauth_api_client.retrieve_registration(user_id, application_id)
     if client_response.was_successful():
       registration_data = client_response.success_response['registration'].get('data')
+      # make sure minprice and maxprice are numbers not strings
+      registration_data['maxprice'], registration_data['minprice'] = int(registration_data['maxprice']), int(registration_data['minprice'])
```

Comment:
Did you submit these changes to the example application? https://github.com/FusionAuth/fusionauth-example-flask-portal/ If not, please do. (It's no fun when a tutorial and example app have different code.)

Reply:
this code is in the example and was just missing from this tutorial